### PR TITLE
John test branch

### PR DESCRIPTION
**Pull request by Thejohnmeyers**
adding pass word conf

### CODE REVIEW

File: src/App.js
Comment (idugan100) [inline src/App.js:1]: I think we changed this to try to let you register for an account and forgot to put it back. Let's undo changes in this file.

File: src/redux/reducers/status.js
Comment (idugan100) [inline src/redux/reducers/status.js:1]: I think we changed this to try to let you register for an account and forgot to put it back. Let's undo changes in this file.

**Review by idugan100 — CHANGES_REQUESTED** [review]
Functionality looks good. Just need to undo changes we did to front end registration logic.

File: src/App.js
Comment (Thejohnmeyers) [reply src/App.js:1]: all good

File: src/redux/reducers/status.js
Comment (Thejohnmeyers) [reply src/redux/reducers/status.js:1]: all good

**Review by idugan100 — APPROVED** [review]

**Pull request merged**